api/src/main/java/org/openmrs/ConceptDatatype.java

```diff
@@ -66,8 +80,17 @@ public class ConceptDatatype extends BaseChangeableOpenmrsMetadata {
 	
 	// Fields
 	@DocumentId
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "concept_datatype_id_seq")
+	@GenericGenerator(
+			name = "concept_datatype_id_seq",
+			strategy = "native",
+			parameters = @Parameter(name = "sequence", value = "concept_datatype_concept_datatype_id_seq")
+		)
+	@Column(name = "concept_datatype_id")
 	private Integer conceptDatatypeId;
 	
+	@Column(name = "hl7_abbreviation", length = 3, nullable = true)
```

Comment:
`hl7_abbreviation` should not be contains any null values, there for `nullable` should be `false`

Reply:
But in hibernate.xml file it's written not-null=false